### PR TITLE
d/aws_route53_resolver_endpoint: Add Route53resolver endpoint datasource

### LINKED ISSUE #8355: Feature Request: aws_route53_resolver_endpoint data source
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I have a use case where I would like to modularize constructing forwarding rules for Route 53 Resolver endpoints across peered VPCs, which requires looking up the allocated ip addresses on a resolver endpoint. It would be nice to be able to allow the consumer of the module to only provide the resolver endpoint ID, rather than having to perform the lookup themselves to extract the IP addresses.

### New or Affected Resource(s)

* data.aws_route53_resolver_endpoint

### Potential Terraform Configuration

```hcl
data "aws_route53_resolver_endpoint" "destination" {
  name = "name-of-route53-resolver-id"
  id = "route53-resolver-endpoint-id"
  arn = "route53-resolver-endpoint-arn"
}
```

### References

* PR that added the corresponding resource: https://github.com/terraform-providers/terraform-provider-aws/pull/6574

### PR DESCRIPTION
**Pull request by robh007**
Route53resolver endpoint data source with filter & ID support

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Fixes #8355 

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
** New Data Source:** `aws_route53_resolver_endpoint`
```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccDataSourceAwsRoute53ResolverEndpoint'
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (118.34s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (295.04s)
...
```

### CODE REVIEW

**Comment by ewbankkit** [comment]
@robh007 Do you have cycles to fix the merge conflicts on this one?

**Comment by robh007** [comment]
@ewbankkit Sure, I should be able to resolve these tomorrow.

**Comment by Omarimcblack** [comment]
@robh007 any chance you could get this conflict sorted? would love to have this in the official provider!

**Comment by ALLFIVE** [comment]
@Omarimcblack Yes sure, I'll try and sort today.

File: aws/data_source_aws_route53_resolver_endpoint.go
Comment (j-nix) [inline aws/data_source_aws_route53_resolver_endpoint.go]: hey @robh007 just a heads up this has now changed to `"github.com/hashicorp/terraform-plugin-sdk/helper/schema"`

**Comment by teamterraform** [comment]
## Notification of Recent and Upcoming Changes to Contributions

Thank you for this contribution! There have been a few recent development changes that affect this pull request. We apologize for the inconvenience, especially if there have been long review delays up until now. Please note that this is automated message from an unmonitored account. See the [FAQ](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/FAQ.md) for additional information on the maintainer team and review prioritization.

If you are unable to complete these updates, please leave a comment for the community and maintainers so someone can potentially continue the work. The maintainers will encourage other contributors to use the existing contribution as the base for additional changes as appropriate. Otherwise, contributions that do not receive updated code or comments from the original contributor may be closed in the future so the maintainers can focus on active items.

For the most up to date information about Terraform AWS Provider development, see the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/CONTRIBUTING.md). Additional technical debt changes can be tracked with the [`technical-debt` label on issues](https://github.com/terraform-providers/terraform-provider-aws/issues?q=is%3Aopen+is%3Aissue+label%3Atechnical-debt).

As part of updating a pull request with these changes, the most current unit testing and linting will run. These may report issues that were not previously reported.

### Action Required: Terraform 0.12 Syntax

Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/8950
Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/14417

Version 3 and later of the Terraform AWS Provider, which all existing contributions would potentially be added, only supports Terraform 0.12 and later. Certain syntax elements of Terraform 0.11 and earlier show deprecation warnings during runs with Terraform 0.12. Documentation and test configurations, such as those including deprecated string interpolations (`some_attribute = "${aws_service_thing.example.id}"`) should be updated to the newer syntax (`some_attribute = aws_service_thing.example.id`). Contribution testing will automatically fail on older syntax in the near future. Please see the referenced issues for additional information.

### Action Required: Terraform Plugin SDK Version 2

Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/14551

The Terraform AWS Provider has been upgraded to the latest version of the Terraform Plugin SDK. Generally, most changes to contributions should only involve updating Go import paths in source code files. Please see the referenced issue for additional information.

### Action Required: Removal of website/aws.erb File

Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/14712

Any changes to the `website/aws.erb` file are no longer necessary and should be removed from this contribution to prevent merge issues in the near future when the file is removed from the repository. Please see the referenced issue for additional information.

### Upcoming Change of Git Branch Naming

Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/14292

Development environments will need their upstream Git branch updated from `master` to `main` in the near future. Please see the referenced issue for additional information and scheduling.

### Upcoming Change of GitHub Organization

Reference: https://github.com/terraform-providers/terraform-provider-aws/issues/14715

This repository will be migrating from https://github.com/terraform-providers/terraform-provider-aws to https://github.com/hashicorp/terraform-provider-aws. No practitioner or developer action is anticipated and most GitHub functionality will automatically redirect to the new location. Go import paths including `terraform-providers` can remain for now. Please see the referenced issue for additional information and scheduling.

**Comment by ewbankkit** [comment]
@robh007 Thanks for all the work on this contribution. Are you able to make all the changes described above for the Plugin SDK v2 migration?
If not, I am happy to add to your work and get this merged.

**Comment by robh007** [comment]
@ewbankkit I’ve updated the code locally, just having an issue with the filter test. Looks like AWS have made some changes to the api. Can’t see why it’s failing at the moment. I’ll push up my changes tomorrow.

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:207]: Spaces not tabs.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  id = "rslvr-in-8g85830108dd4c82b"
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:216]: Spaces not tabs.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  filter {
    name   = "Name"
    values = ["None-Existent-Resource"]
  }
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:200]: Spaces not tabs.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  filter {
    name   = "Name"
    values = [aws_route53_resolver_endpoint.foo.name]
  }

   depends_on = [aws_route53_resolver_endpoint.foo]
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:190]: Alignment.

```suggestion
  tags = {
    Environment = "production"
    Usage       = "original"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:161]: Spaces not tabs.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  id = aws_route53_resolver_endpoint.foo.id
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:156]: Alignment.

```suggestion
  tags = {
    Environment = "production"
    Usage       = "original"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:93]: Alignment.

```suggestion
  vpc_id            = aws_vpc.foo.id
  cidr_block        = cidrsubnet(aws_vpc.foo.cidr_block, 2, 1)
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:133]: Can use [`composeConfig`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md#combining-test-configurations).

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:166]: Can use [`composeConfig`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md#combining-test-configurations) here.

File: aws/data_source_aws_route53_resolver_endpoint.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint.go:80]:
```suggestion
	rID, rIDOk := d.GetOk("resolver_endpoint_id")
```

File: aws/data_source_aws_route53_resolver_endpoint.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint.go:119]:
```suggestion
		d.SetId(aws.StringValue(resolver.Id))
		d.Set("resolver_endpoint_id", resolver.Id)
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:31]:
```suggestion
					resource.TestCheckResourceAttrPair(datasourceName, "id", resourceName, "id"),
					resource.TestCheckResourceAttrPair(datasourceName, "resolver_endpoint_id", resourceName, "id"),
```

File: website/docs/d/route53_resolver_endpoint.html.markdown
Comment (ewbankkit) [inline website/docs/d/route53_resolver_endpoint.html.markdown:19]:
```suggestion
  resolver_endpoint_id = "rslvr-in-1abc2345ef678g91h"
```

File: website/docs/d/route53_resolver_endpoint.html.markdown
Comment (ewbankkit) [inline website/docs/d/route53_resolver_endpoint.html.markdown:34]:
```suggestion
* `resolver_endpoint_id` - (Optional) The ID of the Route53 Resolver Endpoint.
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:78]: Use `testAccAvailableAZsNoOptInConfig`

https://github.com/terraform-providers/terraform-provider-aws/blob/aff071d49120c136dcb4917133b4eaa6512f0482/aws/resource_aws_instance_test.go#L3301-L3312

otherwise the default acceptance test regions may use Local Zone AZs and get errors like

```
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
    data_source_aws_route53_resolver_endpoint_test.go:19: Step 2/2 error: Error running apply: 2020/10/29 10:40:11 [DEBUG] Using modified User-Agent: Terraform/0.12.26 HashiCorp-terraform-exec/0.10.0
        
        Error: error creating Route53 Resolver endpoint: InvalidRequestException: [RSLVR-00419] There are no resolver resources in the Availability Zone containing subnet: "subnet-055bec065a57e028c"
        
        
--- FAIL: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (23.32s)
```

Comment (robh007) [reply aws/data_source_aws_route53_resolver_endpoint_test.go:78]: Thanks @ewbankkit I'll get these resolved today.

**Comment by robh007** [comment]
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (138.65s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (281.56s)
PASS

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:87]: 2 spaces, not 4.

```suggestion
  tags = {
    Name = "tf-acc-r53-resolver-sn1-%d"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:107]: 2 spaces, not 4.

```suggestion
  tags = {
    Name = "tf-acc-r53-resolver-sn3-%d"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:125]: 2 spaces, not 4.

```suggestion
  tags = {
    Name = "tf-acc-r53-resolver-sg2-%d"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:116]: 2 spaces, not 4.

```suggestion
  tags = {
    Name = "tf-acc-r53-resolver-sg1-%d"
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:158]: 2 spaces, not 4.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  resolver_endpoint_id = aws_route53_resolver_endpoint.foo.id
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:192]: 2 spaces, not 4.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
   filter {
     name   = "Name"
     values = [aws_route53_resolver_endpoint.foo.name]
   }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:211]: 2 spaces, not 4.

```suggestion
data "aws_route53_resolver_endpoint" "foo" {
  filter {
    name = "Name"
    values = ["None-Existent-Resource"]
  }
}
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:194]: 2 spaces, not 3.

```suggestion
  depends_on = [aws_route53_resolver_endpoint.foo]
```

Comment (ewbankkit) [reply aws/data_source_aws_route53_resolver_endpoint_test.go:194]: Actually, remove the `depends_on` line otherwise I get:

```
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
    data_source_aws_route53_resolver_endpoint_test.go:47: Step 2/2 error: After applying this test step and performing a `terraform refresh`, the plan was not empty.
        stdout
        
        
        An execution plan has been generated and is shown below.
        Resource actions are indicated with the following symbols:
         <= read (data resources)
        
        Terraform will perform the following actions:
        
          # data.aws_route53_resolver_endpoint.foo will be read during apply
          # (config refers to values not yet known)
         <= data "aws_route53_resolver_endpoint" "foo"  {
              + arn          = (known after apply)
              + direction    = (known after apply)
              + id           = (known after apply)
              + ip_addresses = (known after apply)
              + name         = (known after apply)
              + status       = (known after apply)
              + vpc_id       = (known after apply)
        
              + filter {
                  + name   = "Name"
                  + values = [
                      + "tf-acc-test-6750279126925066157",
                    ]
                }
            }
        
        Plan: 0 to add, 0 to change, 0 to destroy.
--- FAIL: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (243.61s)
```

**Comment by ewbankkit** [comment]
@robh007 Thanks for sticking with this.
Just a few test formatting changes and the removal of `depends_on` and this should be good to go.

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (robh007) [reply aws/data_source_aws_route53_resolver_endpoint_test.go:194]: Thanks for your patients @ewbankkit, when I did this work last year I didn't have a ```depends_on``` within the filter test, but when I came back to pick this up after your initial review the filter test started to fail on the data lookup. Which I found slightly odd. The AWS api had since changed the filter values, I think initially they where ```uppercase```. However when I read the docs both case types are supported. When I added the ```depends_on``` the error went away. I assumed something as changed within terraform at this point & not the provider.   

If I remove the ```depends_on``` I get the following output.
```
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
    data_source_aws_route53_resolver_endpoint_test.go:47: Step 2/2 error: Error running pre-apply plan: 
        Error: Your query returned no results. Please change your search criteria and try again
        
        
--- FAIL: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (16.33s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (117.56s)
FAIL
FAIL    github.com/terraform-providers/terraform-provider-aws/aws       119.311s
FAIL
```

If I leave the ```depends_on``` in the test passes.

```
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccDataSourceAwsRoute53ResolverEndpoint -timeout 120m
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (107.22s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (239.47s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       241.261s
```

Not sure where to go with this now, I noticed the name parameter is optional on the resource so I'm not sure if that causes a potential issue with the initial plan. If I add a second filter using ```SecurityGroupIds``` I don't see the failure.

```
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccDataSourceAwsRoute53ResolverEndpoint -timeout 120m
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (118.19s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (250.14s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       252.067s
```

```
data "aws_route53_resolver_endpoint" "foo" {
    filter {
      name   = "Name"
      values = [aws_route53_resolver_endpoint.foo.name]
    }

    filter {
      name = "SecurityGroupIds"
      values = [aws_security_group.sg1.id, aws_security_group.sg2.id]
    }
}

```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:197]: Alignment.

```suggestion
  filter {
    name   = "SecurityGroupIds"
    values = [aws_security_group.sg1.id, aws_security_group.sg2.id]
  }
```

File: aws/data_source_aws_route53_resolver_endpoint_test.go
Comment (ewbankkit) [inline aws/data_source_aws_route53_resolver_endpoint_test.go:213]: Alignment.

```suggestion
  filter {
    name   = "Name"
    values = ["None-Existent-Resource"]
  }
```

**Review by ewbankkit — APPROVED** [review]
LGTM (once formatting fixes are in).

```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccDataSourceAwsRoute53ResolverEndpoint_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccDataSourceAwsRoute53ResolverEndpoint_ -timeout 120m
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (79.75s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (213.85s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	213.903s

```

**Comment by breathingdust** [comment]
LGTM :rocket: Thanks @robh007 for the contribution!

Verified Acceptance Tests in Commercial (us-west-2)

```console
make testacc TEST=./aws/ TESTARGS='-run=TestAccDataSourceAwsRoute53ResolverEndpoint_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccDataSourceAwsRoute53ResolverEndpoint_ -timeout 120m
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (87.92s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (219.58s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	221.158s
```

Verified Acceptance Tests in GovCloud (us-gov-west-1)

```console
make testacc TEST=./aws/ TESTARGS='-run=TestAccDataSourceAwsRoute53ResolverEndpoint_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccDataSourceAwsRoute53ResolverEndpoint_ -timeout 120m
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== RUN   TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== PAUSE TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Basic
=== CONT  TestAccDataSourceAwsRoute53ResolverEndpoint_Filter
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Basic (71.55s)
--- PASS: TestAccDataSourceAwsRoute53ResolverEndpoint_Filter (174.82s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	176.390s```

**Pull request merged**
